src/neptune/internal/backends/hosted_file_operations.py

```diff
@@ -481,6 +481,9 @@ def _store_response_as_file(
         target_file = destination
 
     total_size = int(response.headers.get("content-length", 0))
+
+    progress_bar = False if total_size < 1024 * 1024 else progress_bar  # less than one chunk
```

Comment:
1. Magic numbers 🛑 
2. Looks like`content-length` may not be provided. If I understand correctly this doesn't mean that the file is empty; rather it's not possible to determine the target size. In such a case we should probably show the progress bar.

Reply:
Done